truckersmp_cli/args.py

```diff
@@ -0,0 +1,237 @@
+        if vdf_is_available:
+            Args.account = get_current_steam_user()
+        if not Args.account:
+            logging.info("Unable to find logged in steam user automatically.")
+            sys.exit("Need the steam account name (-n name) to update.")
+
+    # info
+    logging.info("AppID/GameID: {} ({})".format(Args.steamid, game))
+    logging.info("Game directory: " + Args.gamedir)
+    logging.info("Prefix: " + Args.prefixdir)
+    if Args.proton:
+        logging.info("Proton directory: " + Args.protondir)
+
+
+def create_arg_parser():
+    """Create ArgumentParser for this program."""
+    desc = """
+A simple launcher for TruckersMP to play ATS or ETS2 in multiplayer.
+
+truckersmp-cli allows to download TruckersMP and handles starting TruckersMP
+through Wine while supporting the Windows versions of
+American Truck Simulator and Euro Truck Simulator 2.
+
+The Windows version of Steam should already be able to run in the same
+Wine prefix. The Windows versions of ATS and ETS2 can be installed and updated
+via SteamCMD while all running Steam processes will be stopped
+to prevent Steam from loosing connection. Your Steam password
+and guard code are required by SteamCMD once for this to work.
+
+On Linux it's possible to start TruckersMP through Proton.
+A working native Steam installation is needed for this.
+SteamCMD can use your saved credentials for convenience.
+"""
+    epilog = "Proton AppID list:\n"
+    for k, v in AppId.proton.items():
+        default_mark = ""
+        if k == AppId.proton["default"]:
+            default_mark += " (Default)"
+        if k != "default":
+            epilog += "    Proton {:13}: {:>10}{}\n".format(k, v, default_mark)
+    ap = argparse.ArgumentParser(
+      description=desc, epilog=epilog,
+      formatter_class=argparse.RawDescriptionHelpFormatter)
+    ap.add_argument(
+      "-a", "--ats",
+      help="use American Truck Simulator", action="store_true")
+    ap.add_argument(
+      "-b", "--beta", metavar="VERSION", type=str,
+      help="""set game version to VERSION,
+              useful for downgrading (e.g. "temporary_1_35")""")
+    ap.add_argument(
+      "-d", "--enable-d3d11",
+      help="use Direct3D 11 instead of OpenGL",
+      action="store_true")
+    ap.add_argument(
+      "-e", "--ets2",
+      help="""use Euro Truck Simulator 2
+              [Default if neither ATS or ETS2 are specified] """,
+      action="store_true")
+    ap.add_argument(
+      "-g", "--gamedir", metavar="DIR", type=str,
+      help="""choose a different directory for the game files
+              [Default: $XDG_DATA_HOME/truckersmp-cli/(Game name)/data]""")
+    ap.add_argument(
+      "-i", "--proton-appid", metavar="APPID", type=int,
+      default=AppId.proton[AppId.proton["default"]],
+      help="""choose a different AppID for Proton (Needs an update for changes)
+              [Default:{}] """.format(AppId.proton[AppId.proton["default"]]))
```

Comment:
I missed this too.
```suggestion
              [Default: {}]""".format(AppId.proton[AppId.proton["default"]]))
```
Was the space moved accidentally (or intentionally)?

Reply:
That was my IDE thinking better. I've turned of "formatting on save" before because of this but "formatting on paste" was still on… besides that I'm unsure why it's formatting in the first place because there isn't a formatting prover chosen for python. Anyway, I turned it off, thanks for spotting.